AI.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Artificial Intelligence</title>
+  <script src="js/jquery-3.2.1.min.js"></script>
+  <script src="js/bootstrap.min.js"></script>
+  <link rel="stylesheet" type="text/css" href="Styling.css">
+  <link rel="stylesheet" href="css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="css/bootstrap-grid.min.css">
+  <link rel="stylesheet" type="text/css" href="css/all.min.css">
+</head>
+<body>
+  <header>
+      <nav class="navbar navbar-expand-md navbar-light  col-md-12 col-lg-12 s">
+      <div class="navbar-brand"><i>
+        AI</i>
+      </div>
+      <button type="submit" class=" navbar-toggler " data-toggle="collapse" data-target="#nav"><span class="navbar-toggler-icon"></span></button>
+      <div class="collapse navbar-collapse" id="nav">
+      <ul class="navbar-nav offset-lg-6 offset-md-2 align-items-center">
+        <li class="nav-item ">
+          <a class="nav-link text-dark" href="AI.html">Home</a>
```

Comment:
- [ ] As pointed out by the previous reviewer, please ensure to include the top navbar with social media links so that it matches the original design just like you have done in the about page.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/169647803-145671c9-c857-4be8-8eb2-ca362c1a9724.png)

### Original Design
![image](https://user-images.githubusercontent.com/38088818/169643550-02437a05-7e0f-4403-965c-bca6fe249536.png)